Mathlib/CategoryTheory/Monoidal/Rigid/Basic.lean

```diff
@@ -100,13 +100,23 @@ notation "η_" => ExactPairing.coevaluation
 notation "ε_" => ExactPairing.evaluation
 
 lemma coevaluation_evaluation :
-    (𝟙 Y ⊗ η_ _ _) ≫ (α_ _ _ _).inv ≫ (ε_ X _ ⊗ 𝟙 Y) = (ρ_ Y).hom ≫ (λ_ Y).inv :=
+    (Y ◁ η_ _ _) ≫ (α_ _ _ _).inv ≫ (ε_ X _ ▷ Y) = (ρ_ Y).hom ≫ (λ_ Y).inv :=
   coevaluation_evaluation'
 
 lemma evaluation_coevaluation :
-    (η_ _ _ ⊗ 𝟙 X) ≫ (α_ _ _ _).hom ≫ (𝟙 X ⊗ ε_ _ Y) = (λ_ X).hom ≫ (ρ_ X).inv :=
+    (η_ _ _ ▷ X) ≫ (α_ _ _ _).hom ≫ (X ◁ ε_ _ Y) = (λ_ X).hom ≫ (ρ_ X).inv :=
   evaluation_coevaluation'
 
+-- temporary lemma
```

Comment:
Temporary in what sense?

Reply:
I intended this lemma would be replaced by whiskering version (`coevaluation_evaluation`) in the future after refactoring proofs in this file.

Reply:
We attempted such a refactoring in #8265.